packages/edit-site/src/components/global-styles/block-preview-panel.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+import { BlockPreview } from '@wordpress/block-editor';
+import { getBlockType, getBlockFromExample } from '@wordpress/blocks';
+import { useResizeObserver } from '@wordpress/compose';
+
+// const normalizedWidth = 248;
+const normalizedHeight = 152;
+
+const BlockPreviewPanel = ( { name } ) => {
+	const blockExample = getBlockType( name )?.example;
+	const panelStyles = {
+		width: '100%',
+		height: `${ normalizedHeight }px`,
+		overflow: 'auto',
+	};
+	const [ containerResizeListener, { width: containerWidth } ] =
+		useResizeObserver();
+
+	return ! blockExample ? null : (
+		<div
+			className="edit-site-block-styles-preview__panel"
+			style={ panelStyles }
```

Comment:
None of this styles seem to change at runtime. Can we move them to `style.scss`?

```scss
.edit-site-global-styles__block-preview-panel {
	width: 100%;
	height: 152px;
	overflow: auto;
}
```

Reply:
Moved all the styles to class, except for height because the variable is used twice in JS.

Reply:
Moved the height also to a `scss` variable instead.